src/frontends/pytorch/src/op/bitwise_not.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/logical_not.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_bitwise_not(NodeContext& context) {
```

Comment:
```suggestion
OutputVector translate_bitwise_not(const NodeContext& context) {
```

Reply:
I remember Maxim should have added this. Please check

Reply:
it is still not merged due to segfault issue in tests

Reply:
for now, input mutation does not work if I will use const context